client.go

```diff
@@ -1087,6 +1091,10 @@ func (c *Client[TTx]) InsertManyTx(ctx context.Context, tx TTx, params []InsertM
 			return 0, err
 		}
 
+		if err := c.validateManyInsertOpts(param.InsertOpts); err != nil {
+			return 0, err
+		}
+
```

Comment:
These method bodies are getting to be fairly long for being entirely duplicated, thoughts on extracting everything except the last line of each into a shared unexported method or helper?

Reply:
WFM. Update to include `Client.insertManyParams`.